client/reader/list-gap/index.jsx

```diff
@@ -41,6 +41,9 @@ var Gap = React.createClass( {
 		this.setState( { isFilling: true } );
 		stats.recordAction( 'fill_gap' );
 		stats.recordGaEvent( 'Clicked Fill Gap' );
+		stats.recordTrack( 'calypso_reader_filled_gap', {
+			stream_id: this.props.store.id
```

Comment:
I noticed that this value is a string like `following` - I would have expected an integer given the suffix _id I think. Perhaps just call it `stream` or `stream_name`?

![screen shot 2016-01-06 at 17 37 06](https://cloud.githubusercontent.com/assets/17325/12149549/243376fe-b49c-11e5-8c65-9d5c259fe0cd.png)